src/main/kotlin/racingcar/ui/InputView.kt

```diff
@@ -0,0 +1,25 @@
+package racingcar.ui
+
+object InputView {
+    fun inputValues(): Pair<Int, Int> {
+        val carCount = getNumber(inputCarCount())
+        val moveCount = getNumber(inputMoveCount())
+        return Pair(carCount, moveCount)
+    }
+
+    private fun inputCarCount(): String? {
+        println("자동차 대수는 몇 대인가요?")
+        return readlnOrNull()
+    }
+
+    private fun inputMoveCount(): String? {
+        println("시도할 횟수는 몇 회인가요?")
+        return readlnOrNull()
+    }
+
+    private fun getNumber(input: String?): Int {
+        val number = input?.toIntOrNull()
+        require(number != null) { "입력한 값이 숫자가 아닙니다." }
```

Comment:
`requireNotNull`을 활용해볼 수 있을 것 같아요. 😃
```suggestion
        requireNotNull(number) { "입력한 값이 숫자가 아닙니다." }
```